content/pages/conference/index.rst

```diff
@@ -0,0 +1,18 @@
+===========
+Conference
+===========
+
+:date: 2023-09-28 09:00
+:url: conference
+:save_as: conference/index.html
+
+`conference 2023 <https://conf.sciwork.dev>`__
+================================================
+    
+sciwrok conference 2023 to provide a physical venue to showcase how programming languages are 
+used for scientific and engineering applications and their best practices. In addition 
+to scientific computing and high-performance computing, we also emphasize data management, 
+process, analytics, and visualization.
+
+- Date: December 9-10, 2023
+- Location: Microelectronics and Information Research Center, NYCU
```

Comment:
There should be city and country names for the location.

Reply:
The city and country should be appended after what you wrote.  That is the convention.  I didn't mean to replace the original name.

Reply:
Once this is fixed you may merge and publish.